networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
-   has `"type": "calico-ipam"` then it should be modified to
-   [disable IPv4 assignments and enable IPv6
-   assigments](/reference/cni-plugin/configuration#ipam).
-1. Add the following environment variable to the calico-node DaemonSet in
-   the `calico.yaml` file.
-   
-   ```yaml
-   - name: IP6
-     value: "autodetect"
    ```
-1. For clusters **not** provisioned with kubeadm (see note below), also add the following environment variable to the calico-node DaemonSet in
-   the `calico.yaml` file. Be sure to set the value for `CALICO_IPV6POOL_CIDR`
-   to the desired pool, it should match the `--cluster-cidr` passed to the
-   kube-controller-manager and to kube-proxy.
+       "ipam": {
+           "type": "calico-ipam",
+           "assign_ipv4": "true",
+           "assign_ipv6": "true"
+       },
+   ```
+
+1. Configure IPv6 **IP address autodetection** and **IP pool** by adding the following variable settings to the environment for the `calico-node` container:
+
+   | Variable name | Value |
+   | ------------- | ----- |
+   | `IP6`         | `autodetect` |
+   | `CALICO_IPV6POOL_CIDR` | the same as the IPv6 range you configured as the cluster CIDR to kube-controller-manager and kube-proxy |
+   | `FELIX_IPV6SUPPORT` | `true` |
+
+1. Apply the edited manifest with `kubectl apply -f`.  
+   New pods will get IPv6 addresses as well as IPv4, and can communicate with each other and the outside world over IPv6.
+
+#### Enable IPv6-only, initial install, Kubernetes
+
+To configure IPv6-only support for Kubernetes during {{site.prodname}} installation, follow these steps.
+
+1. Download the appropriate {{site.prodname}} manifest for IPv6 deployment and save it as `calico.yaml`.
+1. Edit the `calico.yaml` ipam section, and modify it to [disable IPv4 assignments and enable IPv6 assignments](/reference/cni-plugin/configuration#ipam).
+   ```
+       "ipam": {
+           "type": "calico-ipam",
+           "assign_ipv4": "true",
+           "assign_ipv6": "true"
+       },
+   ```
+1. Add the following environment variables to the calico-node Daemonset in the `calico.yaml` file.   
+Be sure to set the value for `CALICO_IPV6POOL_CIDR` to the desired IP pool; it should match the `--cluster-cidr` passed to the kube-controller-manager and to kube-proxy.
 
    ```yaml
    - name: CALICO_IPV6POOL_CIDR
      value: "fd20::0/112"
    ```
-
-1. Ensure in the `calico.yaml` file that the environment variable
-   `FELIX_IPV6SUPPORT` is set `true` on the calico-node DaemonSet.
+1. In the `calico.yaml` file, verify that the environment variable `FELIX_IPV6SUPPORT` is set `true` on the calico-node Daemonset.
 1. Apply the `calico.yaml` manifest with `kubectl apply -f calico.yaml`.
 
->**Note**: For clusters provisioned with kubeadm, {{site.prodname}} autodetects the IPv4 and IPv6 pod CIDRs and does not require configuration.
-{: .alert .alert-info}
-
-#### Using only IPv6
+#### Enable IPv6-only, after install, Kubernetes
```

Comment:
Delete section.  I don't believe we support this, and I cannot vouch for it.

Reply:
Deleted.